tests/test_other.py

```diff
@@ -714,6 +714,24 @@ def test_cmake_emscripten_version(self):
 
     run_process([emcmake, 'cmake', path_from_root('tests', 'cmake', 'emscripten_version')])
 
+  def test_nostdincxx(self):
+    path_from_root('tests', 'hello_world.cpp')
+    cmd = [PYTHON, EMCC] + [path_from_root('tests', 'hello_world.cpp')]
+    run_process(cmd) # run once to ensure cache is warmed up
```

Comment:
why do we need to warm up the cache?

Reply:
I copied this over where it was commented as "# run once to ensure binaryen port is all ready".
But I guess you are right none of this stuff should write to stdout anyway. 
Do you know why the stdout comparising is there?  Stdout should always be empty right.  I think I'll remove it.

Reply:
Yes, seems like we can just drop the stdout comparison and the cache warmup.  Right?   Its the include paths on stderr that we are testing in this test, right?